onnxruntime/core/providers/nuphar/scripts/model_editor.py

```diff
@@ -807,7 +1035,9 @@ def parse_arguments():
                         choices=['to_scan',
                                  'opt_inproj',
                                  'gemm_to_matmul',
-                                 'remove_initializers_from_inputs'])
+                                 'remove_initializers_from_inputs',
+                                 'loop_to_scan',
+                                 'save_scan_sub_graph'])
```

Comment:
>save_scan_sub_graph [](start = 34, length = 19)

There's another tool that can do this: [https://github.com/microsoft/onnxruntime/blob/master/tools/python/dump_subgraphs.py](https://github.com/microsoft/onnxruntime/blob/master/tools/python/dump_subgraphs.py)